client/lib/plans/index.js

```diff
@@ -67,7 +73,7 @@ export function getSitePlanSlug( siteID ) {
 }
 
 export function canUpgradeToPlan( planKey, site = sitesList.getSelectedSite() ) {
-	const plan = get( site, [ 'plan', 'expired' ], false ) ? PLAN_FREE : get( site, [ 'plan', 'product_slug' ], PLAN_FREE );
+	const plan = get( site, [ 'plan', 'expired' ], true ) ? PLAN_FREE : get( site, [ 'plan', 'product_slug' ], PLAN_FREE );
```

Comment:
I'm not really sure why I changed this. Maybe it slipped in somehow? Will double-check. Thanks.